### DISCUSSION TITLE
Cloud Storage putFile causes a [storage/object-not-found] No object exists at the desired reference

### DISCUSSION THREAD

**Discussion by marcfrankel** [invertase/react-native-firebase#6160] [Q&A]
Does anyone know why using the Cloud Storage method putFile, which is for putting a file into cloud storage would cause this error to be thrown? I'm not attempting to access the file after upload, although the upload is not successful...

**Comment by ncutixavier** [comment]
the following code works for me
```
Reference reference = FirebaseStorage.instance
        .refFromURL("gs://test-37967.appspot.com")
        .child('property/images/--user-image);
final TaskSnapshot snapshot = await reference.putFile(_file!);
final downloadUrl = await snapshot.ref.getDownloadURL();
print("URL=> $downloadUrl");
```

**Comment by samoray1998** [comment]
i'm getting the same error and still now i'm not knowing why?

**Reply by ncutixavier** [reply]
On my side, I wanted to upload the image on firebase using flutter. So, if it's the same case, you can drop your code here to help for tracking the issue.

**Comment by muhammadasifdotai** [comment]
My this issue:
**[Error: [storage/object-not-found] No object exists at the desired reference.]**

is resolved by adding the latest:
**GoogleService-Info.plist**
file.
